### ISSUE TITLE
Default pixel font-size cannot be changed via device/browser accessibility settings

### ISSUE THREAD

**Issue by dimodi** [telerik/kendo-themes#4090]
**Describe the bug**

Reported in ticket 1585898

The default px font-size cannot be changed via accessibility settings on Android.

The other devices and browsers seem to have settings that can override even pixel font-sizes, even if the settings are not immediately obvious.

**To reproduce**

1. Use an Android device.
2. Change the text size via the display settings or via the Chrome browser settings.
3. See if the changes take effect on a web page with our web components.

**Expected behavior**

The font size inside our component changes.

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material
* theme-tasks

**Affected suites (please remove the unneeded items)**
* Kendo UI for jQuery
* Kendo UI for Angular
* Kendo UI for React
* Kendo UI for Vue
* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* Chrome (did not test on others)

**Build system information (please remove the unneeded items)**
* OS: Android - I tested on a phone, while the customer on a tablet.

**Comment by joneff** [comment]
How important is that? There is an "easy" fix, but it does require changing a lot...

**Comment by dimodi** [comment]
Well, I am aware of only one customer who has reported this, so objectively, it's not that important. On the other hand, there is always the risk that someone will claim we are not accessible on Android.
